### PR TITLE
feat: Create test cases for protocol stats

### LINKED ISSUE #333: [Dashboard] Create test cases for protocol stats
1. Before applying, Read our [Contributor Guidelines](https://github.com/CarmineOptions/derisk-research/blob/master/CONTRIBUTING.md) and [README](https://github.com/CarmineOptions/derisk-research/blob/master/README.md). 
2. Create test cases negative and positive scenarios for next functions, which you can find [here](https://github.com/CarmineOptions/derisk-research/blob/master/apps/dashboard_app/helpers/protocol_stats.py):
- `get_general_stats`
- `get_supply_stats`
- `get_collateral_stats`
- `get_debt_stats`
- `get_utilization_stats`
3. Create new folder `tests` in `dashboard_app` dir

### PR DESCRIPTION
**Pull request by NowGoody**
resolves #333

### CODE REVIEW

**Comment by djeck1432** [comment]
@NowGoody I can't see new commits

**Comment by djeck1432** [comment]
@NowGoody okay, will check it in a few hours again

**Review by djeck1432 — CHANGES_REQUESTED** [review]
those methods so huge and there are a lot of scenarios, but I can see a little option, also I don't see any negative scenarios

**Comment by NowGoody** [comment]
@djeck1432 yea i know i have been have issues with power. appologies for the delay will fix and finalize it all 

> those methods so huge and there are a lot of scenarios, but I can see a little option, also I don't see any negative scenarios

yea i know am currently having technical issue with my pc couldn't fix it yesterday. but am back will complete the above task with more test cases covering relevant edge cases. Appologies for the delay.

**Comment by djeck1432** [comment]
@NowGoody Okay, thank you for letting me know

**Comment by NowGoody** [comment]
gtest

File: apps/dashboard_app/helpers/protocol_stats.py [hunk 1]
```diff
@@ -253,5 +253,7 @@ def get_utilization_stats(
         },
     )
     utilization_columns = [x for x in data.columns if "utilization" in x]
-    data[utilization_columns] = data[utilization_columns].map(lambda x: round(x, 4))
+    # NOTE: This was made so test can pass, something to look into
```
Comment (djeck1432) [inline apps/dashboard_app/helpers/protocol_stats.py:256]: DO not change code not related to your task, please revert it

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Can't see negatives scenarios, could you please add them

**Comment by djeck1432** [comment]
@NowGoody any updates?

**Comment by NowGoody** [comment]
@djeck1432  updating the test cases 3hrs from now appologies for delay

**Comment by djeck1432** [comment]
last 3 hours, after that you will be unassigned @NowGoody

File: apps/dashboard_app/helpers/protocol_stats.py (hunk 1 above)
Comment (NowGoody) [reply apps/dashboard_app/helpers/protocol_stats.py]: am getting an error `AttributeError: 'DataFrame' object has no attribute 'map'. Did you mean: 'max'?`
this error is independent of the input supplied by the test.

`apply()` does exactly what is expected i.e apply the rounding action on every element in the DataFrame

**Comment by NowGoody** [comment]
negtest

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, run pylint command ONLY For `protocol_stats_test.py` and fix all issues
